lib/msal-node/src/client/ClientApplication.ts

```diff
@@ -412,17 +412,18 @@ export abstract class ClientApplication {
             : this.config.auth.azureCloudOptions;
 
         // using null assertion operator as we ensure that all config values have default values in buildConfiguration()
-        this.logger.verbose(
-            `building oauth client configuration with the authority: ${authority}`,
-            requestCorrelationId
-        );
         const discoveredAuthority = await this.createAuthority(
             authority,
             azureRegionConfiguration,
             requestCorrelationId,
             userAzureCloudOptions
         );
 
+        this.logger.info(
```

Comment:
Why did you change this from verbose to info? Additionally, why did you move this log to after the createAuthority call?

Reply:
There are some many bugs / incidents where the authority that MSAL computes is wrong, it's worth having this at info level.

Reply:
Also, putting this after the discovery will give us the updated authority with the regional environment if it is available.